### PR TITLE
3.  NSSecureCoding implementation for FIRInstallationsStoredItem

### PR DESCRIPTION
**Pull request by maksymmalyhin**

### CODE REVIEW

File: FirebaseInstallations/Source/Tests/Utils/FIRKeydArchivingUtils.m [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ * limitations under the License.
+ */
+
+#import "FIRKeydArchivingUtils.h"
+
+@implementation FIRKeydArchivingUtils
+
++ (nullable NSData *)archivedDataWithRootObject:(id)object error:(NSError **)outError {
+  NSData *archivedData;
+  if (@available(macOS 10.13, iOS 11.0, tvOS 11.0, *)) {
```
Comment (chliangGoogle) [inline FirebaseInstallations/Source/Tests/Utils/FIRKeydArchivingUtils.m]: Should we just start with higher os support considering this is a new SDK?

Comment (maksymmalyhin) [reply FirebaseInstallations/Source/Tests/Utils/FIRKeydArchivingUtils.m]: I'll be happy to get rid of such conditions if it is an option.
@ryanwilson @paulb777 WDYT?

Comment (chliangGoogle) [reply FirebaseInstallations/Source/Tests/Utils/FIRKeydArchivingUtils.m]: although we might need to cover the minimum iOS version for Firebase overall, which is iOS 8? In this case, maybe we still need to keep this logic?

Comment (maksymmalyhin) [reply FirebaseInstallations/Source/Tests/Utils/FIRKeydArchivingUtils.m]: so far, yes, this is the main motivation. FCM and Analytics (the main clients of FIS) support iOS 8 and higher

Comment (chliangGoogle) [reply FirebaseInstallations/Source/Tests/Utils/FIRKeydArchivingUtils.m]: Let's keep it then.

**Pull request merged**